src/utils.py

```diff
@@ -0,0 +1,153 @@
+    if hover_data and show_red:
+        idx = hover_data['points'][0]['pointIndex']
+        redundancy_data = data.get_redundancy()
+        # Protect against case where the last hovered point was from a larger data set.
+        if idx < data.n:
+            # Note: this modifies global data and so is not compatible with use
+            # of the demo in a multi-user environment.  One alternative is to
+            # make a copy of the DataFrame prior to modifying the redundancy
+            # column.
+            df['Redundancy'] = redundancy_data[idx]
+            hover_cols['Redundancy'] = False
+
+        color_data = df['Redundancy'].values
+        normalized_color_data = (color_data - np.min(color_data)) / (np.max(color_data) - np.min(color_data))
+        rgba_colors = sample_colorscale(COLOR_SCALE, normalized_color_data, colortype='rgb')
+
+        for i in range(len(rgba_colors)):
+            rgba_colors[i] = 'rgba' + rgba_colors[i][3:-1] + ', ' + str(opacity[i]) + ')'
+
+        if data.n < 30:
+            display_text = [round(i.item(),2) for i in color_data]
+    else:
+        rgba_colors = [f'rgba(42, 125, 225, {o})' for o in opacity]
+        
+    # Plot the bar graph
+    fig = go.Figure(data=[go.Bar(
+        x=df["Feature"], 
+        y=df["Feature Relevance"], 
+        text=display_text,
+        textposition='outside',
+    )])
+
+    fig.update_traces(
+        marker_color=rgba_colors,
+        marker_line_color='black',
+        marker_line_width=mlw,
+        hoverinfo='none', 
+        hovertemplate=None,
+    )
+    
+    fig.update_layout(
+        font=dict(size=GRAPH_FONT_SIZE),
+        yaxis_title='Feature Relevance to Outcome',
+        yaxis_range=[0,1.1],
+    )
+    
+    # Modify axis labels:
+    if data.name == 'titanic':
+        fig.update_layout(xaxis_title='Passenger Features')
+    elif data.name == 'scene':
+        fig.update_layout(xaxis_title='Color and Texture Features in Image')
+
+    return fig
+
+def draw_accuracy_bars(data: DataSet, selected_features: list, soln_score: float) -> go.Figure:
+    """Draws the accuracy bar chart for output.
+
+    Args:
+        data: The DataSet object for the given data set.
+        selected_features: Solution (if available). If not available then None.
+        soln_score: Accuracy score for model using selected_features.
+
+    Returns:
+        go.Figure: A Plotly figure object.
+    
+    """
+
+    scores = [round(data.baseline_cv_score,2), round(soln_score,2)]
+
+    fig = go.Figure(data=[go.Bar(
+        x=[str(data.n), str(len(selected_features))], 
+        y=scores, 
+        text=scores,
+        textposition='outside',
+    )])
+
+    fig.update_traces(
+        marker_color=[COLOR_SCALE[1], COLOR_SCALE[-1]],
+        marker_line_color='black',
+        marker_line_width=1,
+        hoverinfo='none', 
+        hovertemplate=None,
+    )
+
+    fig.update_layout(
+        font=dict(size=GRAPH_FONT_SIZE),
```

Comment:
This will make the graph bigger

```suggestion
        font=dict(size=GRAPH_FONT_SIZE),
        margin={"t": 0, "l": 0, "b": 0, "r": 0},
```